src/main/java/io/millesabords/mongo2arango/BulkInsert.java

```diff
@@ -48,28 +50,31 @@ public void modify(DBCursor cursor) {
                     }
                 })
                 .as(Map.class);
-
-        arango.startBatchMode();
-
-        String id;
+        
         int total = 0;
         int count = 0;
         Map doc;
+        final String arangoDatabaseName = cfg.get(Config.ARANGO_DB);
         final String arangoColl = cfg.get(Config.ARANGO_COLLECTION);
         final int batchSize = cfg.getInt(Config.ARANGO_BATCH_SIZE);
-
+        
+        ArangoCollection arangoCollection = arango.db(DbName.of(arangoDatabaseName)).collection(arangoColl);
+		if (!arangoCollection.exists()) {
+			LOGGER.info("ArangoDB | Creating collection : {}", arangoCollection.name());
+			arangoCollection.create();
+		}
+        
         while (cursor.hasNext()) {
             total++;
             count++;
             doc = cursor.next();
-            id = doc.remove("_id").toString();
+            doc.remove("_id").toString();
```

Comment:
I think you can remove the "toString()" call

Reply:
Yes, forgot about that.

Reply:
I removed it from my code but i'll wait to submit a new pull request when the features to also convert Arango to Mongo and also check foreign keys are finished.